devicetypes/smartthings/zigbee-motion-sensor-light.src/led-cpx-light.groovy

```diff
@@ -65,24 +65,29 @@ private getMOTION_CLUSTER() { 0x0406 }
 private getMOTION_STATUS_ATTRIBUTE() { 0x0000 }
 private getON_OFF_CLUSTER_VALUE() { 0x0006 }
 private getCONFIGURE_REPORTING_RESPONSE() { 0x07 }
+private getON_DATA_VALUE() { [0x01, 0x00] }
+private getOFF_DATA_VALUE() { [0x00, 0x00] }
 
 def parse(String description) {
 	def event = zigbee.getEvent(description)
+	def zigbeeMap = zigbee.parseDescriptionAsMap(description)
 	
 	if (event) {
+		if (zigbeeMap.clusterInt == ON_OFF_CLUSTER_VALUE && (zigbeeMap.data != ON_DATA_VALUE || zigbeeMap.data != OFF_DATA_VALUE)) {
+			return
+		}
```

Comment:
Is there any particular reason that we need to check whole array? Wouldn't it work if only data[0] was checked?

Reply:
Oh, I didn't thought that. OK, I'll change that.